app/models/sample_manifest/library_tube_behaviour.rb

```diff
@@ -17,6 +17,10 @@ def initialize(manifest)
       @tubes = []
     end
 
+    def generate
+      @tubes = generate_tubes(Tube::Purpose.standard_library_tube)
+    end
+
```

Comment:
Moved this up to maintain the same organization with the other behaviour classes.